manifests/zookeeper/README.md

```diff
@@ -0,0 +1,9 @@
+# How to install: 
+# Just execute by this way
+
+kubectl create -f namespace.yaml
+kubectl create -f zk.headless.svc.yaml
+kubectl create -f zk.svc.yaml
```

Comment:
slight lack of consistency in the naming here. Either you use . dots all the time or you use dashes for names then a dot for suffic.

`zk-headless-svc.yaml`